src/graphql.rs

```diff
@@ -3641,22 +3684,33 @@ impl ___Type for FilterEntityType {
                         not_column_exists = true;
                     }
 
-                    match utype.unmodified_type() {
-                        __Type::Scalar(s) => Some(__InputValue {
+                    match utype.nullable_type() {
```

Comment:
Now that we support filtering by array columns, we want to respect the fact that the `utype` is an array, so we'll only unwrap Non-Nullables.